source/administration/mmctl-cli-tool.rst

```diff
@@ -1582,6 +1584,76 @@ mmctl config get
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
 
+mmctl config migrate
+^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+Migrates a file-based configuration to (or from) a database-based configuration. Point the Mattermost server at the target configuration to start using it.
+
+**Format**
+
+.. code-block:: sh
+
+   mmctl config migrate [from_config] [to_config] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+   config migrate path/to/config.json "postgres://mmuser:mostest@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"
+
+**Options**
+
+.. code-block:: sh
+
+   -h, --help   help for migrate
+
+**Options inherited from parent commands**
+
+.. code-block:: sh
+
+   --config-path string           path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")
+   --format string                the format of the command output [plain, json] (default "plain")
+   --insecure-sha1-intermediate   allows to use insecure TLS protocols, such as SHA-1
+   --local                        allows communicating with the server through a unix socket
+   --strict                       will only run commands if the mmctl version matches the server one
+
+mmctl config reload
+^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+Reloads the server configuration in case you want new settings to be applied.
```

Comment:
```suggestion
Reloads the server configuration and applies new settings.
```

Reply:
Is my interpretation correct? If so, my suggested wording isn't ideal and is open to refinement. :)